serverless.yml

```diff
@@ -0,0 +1,21 @@
+service: awsWiki
```

Comment:
Why do we need this file?

Reply:
The deletion didn't get pushed up, just pushed. Sorry :(